### PR TITLE
PHPCS-12: open tag

### PR DESCRIPTION
**Pull request by bestit-lubisch**

### CODE REVIEW

**Comment by bestit-tkellner** [comment]
I think this sniff checks more as in requirements specified.

**Comment by bestit-lubisch** [comment]
IMO the check "OpenTagNotFirstStatement" is the only one which is more than expected.
What should i do with this check?

**Comment by bestit-tkellner** [comment]
I think one check per sniff.

**Comment by bestit-lubisch** [comment]
This is too general. In cases like this one, its preferable to ensure a line after the open tag und additionally check if its empty. 
If i put every check in a separate sniff it would produce much code duplication for no reason.

**Comment by bestit-tkellner** [comment]
I think if we abstract our classes we don't create much duplicate code.

**Comment by bestit-tkellner** [comment]
Rename to open tag sniff.

**Comment by bestit-lubisch** [comment]
done.

File: README.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ Execute the PHP_CodeSniffer (path can vary on your composer configuration):
 | Generic.Formatting.SpaceAfterCast | BestIt | There MUST be a space after cast. |
 | Generic.Files.LineLength | PSR-2 | Every line MUST not be longer than 120 chars. |
 | Generic.Arrays.DisallowLongArraySyntax | BestIt | Every array syntax MUST be in short array syntax. |
+| BestIt.Formatting.EmptyLineAfterOpenTagSniff | BestIt | After the open tag there MUST be an empty line. |
```
Comment (bestit-tkellner) [inline README.md]: Please correct this line.

**Comment by bestit-lubisch** [comment]
fixed this line.

**Pull request merged**